### DISCUSSION TITLE
How to specify Istio Gateway Kind, to deploy istio ingress to specific kube nodes based on selectors

### DISCUSSION THREAD

**Discussion by Gullimama** [istio/istio#50836] [Q&A]
Hi Team, 

We have a Kind: Gateway which is applied via terraform. How to make sure the istio deployment that is created will have node affinity so the all pods are deployed on specific kuberenetes node pool based on a poolname selector?

**Comment by husnialhamdani** [comment]
I think you only need to add nodeSelector https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/ to the Gateway deployment config.

**Reply by husnialhamdani** [reply]
Is that answered your question?
